ql/src/test/org/apache/hadoop/hive/ql/txn/compactor/CompactorTest.java

```diff
@@ -129,6 +130,7 @@ public abstract class CompactorTest {
   FileSystem fs;
   
   @Before
+  @BeforeEach
```

Comment:
Junit 4 & Junit 5 tests are present in the compaction related tests. Hence have to use both the annotations. I think its best to address moving compaction related tests from Junit 4 to Junit 5 tests in a separate Jira since this PR does not have such intentions to change it.